frontend/src/commit-panel/Notice.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+import { DetailsLevel } from '../enums/enums';
+
+interface CommitPanelNoticeProps {
+  detailsLevel: DetailsLevel;
+  onDetailsLevelChange(detailsLevel: DetailsLevel): void;
+}
+
+export default class Notice extends React.Component<CommitPanelNoticeProps, {}> {
+
+  onDetailsClick = (e: React.MouseEvent) => {
+    e.preventDefault();
+
+    const { detailsLevel, onDetailsLevelChange } = this.props;
+    onDetailsLevelChange(detailsLevel === DetailsLevel.None ? DetailsLevel.Overview : DetailsLevel.None);
+  };
+
+  render() {
+    const { detailsLevel } = this.props;
+
+    return (
+      <p>
+        {`You have `}
+        <a
+          href='http://docs.versionpress.net/en/feature-focus/undo-and-rollback#uncommitted-files'
+          target='_blank'
+        >
+          uncommitted changes
```

Comment:
I would prefer more compact style (text on the same line) when it is short (as discussed [here](https://github.com/versionpress/versionpress/issues/1086#issuecomment-231018574)).
In this case:
```jsx
<a
  href='http://docs.versionpress.net/en/feature-focus/undo-and-rollback#uncommitted-files'
  target='_blank'
>uncommitted changes</a>